tests/test-cmd/test.run.js

```diff
@@ -308,7 +308,7 @@ describe('run', () => {
         addonId: 'some-addon@test-suite',
         client,
         // $FLOW_IGNORE: fake a Firefox ChildProcess using an EventEmitter for testing reasons.
-        firefox: new EventEmitter(),
+        firefoxApp: new EventEmitter(),
```

Comment:
This is an EventEmitter used as a fake ChildProcess for testing reasons, it should be renamed to `firefoxProcess`, for the same reasons detailed in the above comment.